osstracker-docs/README.md

```diff
@@ -7,3 +7,77 @@ High Level View
 ===============
 
 ![High Level View](https://raw.githubusercontent.com/Netflix/osstracker/documentation/osstracker-docs/HighLevelView.png)
+
+Manual installation
+===================
+
+Following manual installation is sample installation based on Linux system using docker containers. You can adapt it easily for other system using *docker-machine*, but care about `CASS_HOST` and `ES_HOST` they should be equals to `docker-machine ip MACHINE_NAME`.
+
+**ATTENTION** neither *Cassandra* nor *ElasticSearch* containers define mounting volumes, if containers are removed data will be lost.
+
+Clone OSS Tracker source code
+
+```
+git clone https://github.com/Netflix/osstracker.git
+```
+
+Run Cassandra instance (using docker)
+
+```
+export CASS_HOST=localhost
+export CASS_PORT=9042
+docker run -d --name osstracker-cassandra -p ${CASS_PORT}:9042 cassandra
+```
+
+Init Cassandra DDL
+
+```
+docker run -it --link osstracker-cassandra:cassandra -v $(pwd)/osstracker-ddl/:/tmp/ddl/ --rm cassandra cqlsh cassandra -f /tmp/ddl/osstracker-simple.cql
+```
+
+Run ElasticSearch instance (using docker)
+
+```
+export ES_HOST=localhost
+export ES_PORT=9200
+docker run -d --name osstracker-elasticsearch -p ${ES_PORT}:9200 elasticsearch
+```
+
+Init ElasticSearch DDL
+
+```
+curl -i -X PUT ${ES_HOST}:${ES_PORT}/osstracker
+curl -i -X PUT -H "Content-Type: application/json" -d '{"properties":{"repo_name":{"type":"string","index":"not_analyzed"}}}' ${ES_HOST}:${ES_PORT}/osstracker/_mapping/repo_stats
+```
```

Comment:
Yes I write installation for Linux env (without *docker-machine*) but as I explain at start of the guide https://github.com/Netflix/osstracker/pull/10/files#diff-bc74d8ae8862e6340b2c5165f8a88fb4R14 you should replace `ES_HOST` and `CASS_HOST` by `docker-machine ip <MACHINE NAME>`

But if you're not using `docker-machine` (or `boot2docker`) yes you need to find your private network of your virtual machine.

BTW I wrote this guide for linux but it based on Mac OSX installation I done :)